### PR TITLE
Added SVM model with 98% accuracy

### LINKED ISSUE #165: Creating a SVM for achieving higher accouracy (98%)
Several methods show that ML techniques can show high accuracy levels ranging rom 90%- 97%. Since we have used RFR in the main branch, I'll be creating a SVM to show much higher accuracy.
Also, I'll be improving the model by normalizing the training data which is a scaling feature process. I'll be further searching hyperparameters for improving the model. 

I want to work on this issue under GSSoC 22.

### PR DESCRIPTION
**Pull request by theshredbox**
# Description

I have created a SVM model that has an accuracy of 98% which is impressively high for a ML Model. Also, I have compared this SVM model to a Grid Search Model to prove that my chosen model is the best.

Fixes #165 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

The model has been tested using the best search hyperparamters. The model has also been compared to a gridsearching model to challenge it's accuracy. Heat map comparisons have been visualized for the same.

image

image

image

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by freny24** [comment]
@theshredbox  Excellent work ! 👏🙌

**Review by srimanikantapalakollu — CHANGES_REQUESTED** [review]
Jupyter Notebooks are Not required for this project. Any Model implemented should be able to integrate it into the Application.

**Comment by freny24** [comment]
@theshredbox Develop a python script so that it easily integrates into flask. Jupyter notebooks are not preferred! You can add your python script in this same PR.
